### PR TITLE
made small changes to the readme file

### PR DESCRIPTION
**Pull request by regan008**
small changes

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -1 +1,2 @@
 Hello World!
+**THESE ARE SOME CHANGES**
```
Comment (zuoweitan) [inline README:2]: some changes made by zuoweitan

**Comment by coddamn** [comment]
help I'm new to Git !!

**Comment by saifulhasib** [comment]
i am new.pls help me

**Comment by coderhh** [comment]
LOL

**Comment by ghost** [comment]
This is worse than command line in Linux!

**Comment by ghost** [comment]
Anyone a Python crypto nut?

**Comment by w-n-er** [comment]
how to use this?

**Comment by ghost** [comment]
Enter at your own risk!

**Comment by ethomas01** [comment]
how to use github
i dont know wher to submit my code

**Pull request closed without merging**